fmralign/tests/utils.py

```diff
@@ -0,0 +1,92 @@
+from sklearn.utils.testing import assert_array_almost_equal, assert_greater
+from sklearn.metrics import r2_score
+from nilearn.input_data import NiftiMasker
+import numpy as np
+import nibabel
+
+
+def assert_class_align_better_than_identity(algo, X, Y):
+    """ Tests that the given algorithm align ndarrays X into Y better than identity. This alignment is measured through r2 score.
+    """
+    print(algo)
+    algo.fit(X, Y)
+    identity_baseline_score = zero_mean_coefficient_determination(
+        Y, X)
+    algo_score = zero_mean_coefficient_determination(Y, algo.transform(X))
+    assert_greater(algo_score, identity_baseline_score)
+
+
+def assert_algo_transform_almost_exactly(algo, img1, img2, mask=None):
+    """ Tests that the given algorithm manage to transform almost exactly Nifti image img1 into Nifti Image img2
+    """
+    algo.fit(img1, img2)
+    imtest = algo.transform(img1)
+    masker = NiftiMasker(mask_img=mask)
+    masker.fit()
+    assert_array_almost_equal(masker.transform(
+        img2), masker.transform(imtest), decimal=6)
+
+
+def random_nifti(shape):
```

Comment:
random_niimg in Nilearn jargon

Reply:
in test_alignment_methods.py, it's the test called : test_all_classes_better_than_identity()